pkg/renderer/templates/public_values.yaml.tpl

```diff
@@ -201,6 +201,7 @@ scannerV4:
 {{- $_ := unset $envVars "ROX_OFFLINE_MODE" -}}
 {{- $_ := unset $envVars "ROX_TELEMETRY_ENDPOINT" -}}
 {{- $_ := unset $envVars "ROX_TELEMETRY_STORAGE_KEY_V1" -}}
+{{- $_ := unset $envVars "ROX_SCANNER_V4" -}}
```

Comment:
Another option would be to rename the CI flag to e.g. `CENTRAL_SCANNER_V4_SUPPORT` so that it's not identical to the name of the feature flag. Then this line won't be needed, and should also fix the issue.

Reply:
OTOH, I believe that importing feature flags from the environment probably exists so that we can test features that are not yet added to the Helm chart.

Since Scanner-V4 can be enabled with the feature flag, also the solution above in valid IMO.